fabric-network-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-network-v1",
```

Comment:
Should this be networking? Network sounds a bit odd to me at least since it describes a network provided by the fabric api when it really is a networking helper,

Reply:
I don't see how this is resolved.

Reply:
This is network provided by fabric api. The vanilla game doesn't have custom payload packet registration, login query hooks, and all similar features. This is a complete set of hooks (not just some utilities), and is not just a "helper".

Reply:
If you have other concerns about this, feel free to navigate to this file and start a new conversation.